source/common/config/grpc_mux_impl.cc

```diff
@@ -23,7 +23,8 @@ GrpcMuxImpl::GrpcMuxImpl(const LocalInfo::LocalInfo& local_info, Grpc::AsyncClie
 GrpcMuxImpl::~GrpcMuxImpl() {
   for (const auto& api_state : api_state_) {
     for (auto watch : api_state.second.watches_) {
-      watch->inserted_ = false;
+      // watch->inserted_ = false;
```

Comment:
@eziskind can you do a follow up to clean up this comment? Thank you.

Reply:
Woops - will clean that up. Thanks for catch!